src/Sylius/Bundle/UiBundle/Resources/assets/less/AdminLTE/skin-sylius.less

```diff
@@ -0,0 +1,71 @@
+@import "node_modules/bootstrap/less/mixins.less";
```

Comment:
why not makes this non-publishable-into-web-folder files lives in `Resources/private/less/...` in opposit to the `public` folder which will be published as assets in web ?